boards/arm/mimxrt1024_evk/mimxrt1024_evk.yaml

```diff
@@ -24,3 +24,4 @@ supported:
   - sdhc
   - adc
   - usb_device
+  - flexpwm
```

Comment:
this should be `- pwm` to trigger PWM testcases: https://github.com/zephyrproject-rtos/zephyr/blob/15c77acfb4326b7c39f3d2d48f13fbd9fe30404e/tests/drivers/pwm/pwm_api/testcase.yaml#L9

Reply:
Done!